lib/views/header-bar.js

```diff
@@ -76,38 +74,32 @@ class HeaderBarView extends React.Component {
 
   renderRight() {
     return <View style={styles.right}>
+      <TouchableOpacity
```

Comment:
Perf problem now fixed. It was related to HTTP requests blocking JS thread during animation.